app/src/main/java/com/xmartlabs/scasas/doapp/ui/OnBoardingActivity.java

```diff
@@ -33,12 +41,17 @@ protected void onCreate(Bundle savedInstanceState) {
     showNavigationControls(false);
     setOnboardPages(pages);
     setFinishButtonTitle(R.string.lets_do_it);
-    cardViewPager.setPadding(75, 276, 75, 281);
+    cardViewPager.setPadding(75, 86, 75, 91);
```

Comment:
dont use magic numbers! use constants and also take care of the unity! use the metric helper